docs-ui/components/sample.tsx

```diff
@@ -1,12 +1,79 @@
+import {createContext, ReactChild, useState} from 'react';
+import {ThemeProvider, useTheme} from '@emotion/react';
 import styled from '@emotion/styled';
 
+import {IconMoon} from 'app/icons';
 import space from 'app/styles/space';
+import {darkTheme, lightTheme, Theme} from 'app/utils/theme';
 
-const Sample = styled('div')`
+type ThemeName = 'dark' | 'light';
+
+type Props = {
+  children?: ReactChild;
+  showThemeSwitcher?: boolean;
+  noBorder?: boolean;
```

Comment:
Might be nice to add some `/** */` comments here

Reply:
Good point